package.json

```diff
@@ -149,7 +149,7 @@
     "eslint-plugin-jest": "22.4.1",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.21.5",
+    "eslint-plugin-react": "7.28.0",
```

Comment:
This update is required because the react eslint plugin added extra fields that the sdl rules configure, so an older eslint-plugin-react makes eslint fail with a config error...

Reply:
I hope the pull-bot is just overprotective and this is okay since it is a devdependency.